src/image_copy/main.go

```diff
@@ -67,21 +70,68 @@ func getTags(dstTag string, updateMajor bool) []string {
 	return tags
 }
 
+func verifyImage(imageTag, platform, check string) {
```

Comment:
You should rename `check` to something like `expectedArchitecture` as that's what we're looking for